.update/patches

```diff
@@ -1480,6 +1480,10 @@ Patch_8_23()
 		G_EXEC_OUTPUT=1 G_EXEC dpkg -i --force-confdef,confold package.deb
 		G_EXEC rm package.deb
 	fi
+	# ADS-B Feeder used to install two service files that aren't needed
+	# the names should be specific enough to make an unconditional force remove acceptable
+	G_EXEC rm -f /etc/systemd/system/adsb-update.service
+	G_EXEC rm -f /etc/systemd/system/adsb-update.timer
```

Comment:
I wasn't sure if this is the preferred way to do this - feel free to point me at a better approach

Reply:
Let's be conservative and only touch things if it was really installed via `dietpi-software`. And disable those, if enabled.
```sh
	# ADS-B Feeder used to install two service files that aren't needed: https://github.com/MichaIng/DietPi/pull/6661
	if [[ -f '/boot/dietpi/.installed' ]] && grep -q '^[[:blank:]]*aSOFTWARE_INSTALL_STATE\[141\]=2' /boot/dietpi/.installed
	then
		systemctl -q is-enabled adsb-update.timer && G_EXEC systemctl --no-reload disable --now adsb-update.timer
		[[ -f '/etc/systemd/system/adsb-update.timer' ]] && G_EXEC rm /etc/systemd/system/adsb-update.timer
		systemctl -q is-enabled adsb-update && G_EXEC systemctl --no-reload disable --now adsb-update
		[[ -f '/etc/systemd/system/adsb-update.service' ]] && G_EXEC rm /etc/systemd/system/adsb-update.service
	fi
```

Reply:
I didn't know that `disable` also supports a `--now` flag... I thought this one was asymmetric for some strange reason... and of course I have no idea what you are actually doing with the grep on `/boot/dietpi/.installed` ... something for me to study a bit, I guess 🙃

Reply:
The grep checks whether the software title with ID 141 is installed (`=2`).